src/style.css

```diff
@@ -0,0 +1,113 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+} */
+
+/* html {
+  font-family: "Avenir", Arial, sans-serif;
+} */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300px;
+}
+
+.form-big {
+  position: relative;
+  min-width: 200vh;
+  margin-top: 20px;
+  margin-right: 8px;
+  margin-left: 8px;
```

Comment:
why do we need this margins? Just not reset margins on body

Reply:
I use this margin to center my search and pass tests